sources/packages/web/src/components/common/StudentApplication.vue

```diff
@@ -0,0 +1,273 @@
+<template>
+  <formio
+    :formName="selectedForm"
+    :data="initialData"
+    :readOnly="isReadOnly"
+    @loaded="formLoaded"
+    @changed="formChanged"
+    @submitted="submitted"
+    @customEvent="customEvent"
+  ></formio>
+  <StudentApplicationCommonActions
```

Comment:
I still do not feel the need of creating this component. Is it going to grow or have much more logic that justifies it?